app/models/chargeback_rate_detail_measure.rb

```diff
@@ -0,0 +1,19 @@
+class ChargebackRateDetailMeasure < ActiveRecord::Base
+  validates :name, :presence => true, :length => {:maximum => 100}
+  validates :step, :presence => true, :numericality => {:greater_than => 0}
+  validates :units, :presence => true, :length => {:minimum => 2}
+  validates :units_display, :presence => true, :length => {:minimum => 2}
+  validate :units_same_lenght
+
+  has_many :chargeback_rate_detail
+
+  def units_same_lenght
+    unless (units.count == units_display.count)
+      errors.add("Units Problem", "Units_display lenght diferent that the units lenght")
+    end
+  end
+
+  def measures_for_select
```

Comment:
I'm assuming this is mainly for the UI, but it's also used in reverse in ChargebackRateDetail#per_unit_display.  I'm thinking that there should be a method called `measures` or something that does `units.zip(units_display).to_h`, and then `measures_for_select` should be `measures.invert` (or just don't have the `measures_for_select` method at all and put the invert directly in the UI code.  This way the "real" method used by the models is the main method, and the UI is just a transformation of that.